Src/BootCamp.Chapter/MacFactory.cs

```diff
@@ -7,9 +7,24 @@ namespace BootCamp.Chapter
 {
     public class MacFactory
     {
+        private DesktopComputer _macComputer;
+        private string manufacturersId;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'manufacturersId' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5121609)